tests/test_ilastik/helpers/shellGuiTestCaseBase.py

```diff
@@ -150,12 +174,26 @@ def getPixelColor(self, imgView, coordinates, debugFileName=None, relativeToCent
 
         return img.pixel(point)
 
-    def moveMouseFromCenter(self, imgView, coords, modifier=Qt.NoModifier):
-        centerPoint = old_div(imgView.rect().bottomRight(), 2)
-        point = QPoint(*coords) + centerPoint
+    def moveMouseFromCenter(
```

Comment:
The comment around the sole usage of this method says: ["Move the cursor out of the way so we can sample the image"](https://github.com/k-dominik/ilastik/blob/e01f1f5962ed0bf7369bbf529259fc567ca0f135/tests/test_ilastik/test_applets/pixelClassification/testPixelClassificationGui.py#L613). Now we can just hide the cursor, do we need this method at all?

Reply:
You mean for the `movemouseFromCenter`, you'd always want to hide the cursor?

Reply:
> "Move the cursor out of the way so we can sample the image"

[At the call site](https://github.com/k-dominik/ilastik/blob/e01f1f5962ed0bf7369bbf529259fc567ca0f135/tests/test_ilastik/test_applets/pixelClassification/testPixelClassificationGui.py#L613), it seems the mouse is moved out of the way because the crosshair interferes with image sampling, exactly the same problem you fixed in this PR.

Reply:
yeah, that's why I removed all calls to the `moveMouseFromCenter` method. Because it's not adequate for the use case here... But I can imagine other use-cases that would not necessarily involve hiding the cursor, but actually moving it.